plugins/@grouparoo/sqlite/src/lib/export/exportRecord.ts

```diff
@@ -98,9 +107,12 @@ export const exportRecord: ExportRecordPluginMethod = async ({
     // --- Groups --- //
 
     // delete existing groups
-    const deleteGroupsQuery = `DELETE FROM "${groupsTable}" WHERE ${groupForeignKey} = ${newRecordProperties[primaryKey]}`;
-    validateQuery(deleteGroupsQuery);
-    await connection.asyncQuery(deleteGroupsQuery);
+    await connection.asyncQuery(
+      ...SQLiteQueryBuilder.build(
+        `DELETE FROM "${groupsTable}" WHERE ${groupForeignKey} = ?`,
```

Comment:
might as well make all interpolations a `?`, so the args would be `[groupsTable, newRecordProperties[primaryKey]`

Reply:
I tried, but it's not how the way this version of the module's parser works. I got an error when I tried. It seems that it only parses values but not the table names for some reason. I'll dig a little deeper to see if this was a mistake.

Reply:
I did another try and it failed. It looks like the implementation of the library goes all the way down the sqlite C implementation and looks like not all values can be substituted:

https://stackoverflow.com/a/597198/20494